### ISSUE TITLE
How to show losses of all samples in a batch

### ISSUE THREAD

**Issue by ccx1997** [SeanNaren/warp-ctc#169]
I want the ctc loss_fn to return a tensor (with shape as [batch]) containing every loss in a batch, like the parameter `reduction` of CTCloss in Pytorch: 
`ctcloss = torch.nn.CTCLoss(reduction='none')`
How can I do?
